src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -194,12 +198,15 @@ static ucs_status_t uct_ib_mlx5_devx_reg_ksm_data_addr(
 
 static ucs_status_t
 uct_ib_mlx5_devx_reg_ksm_data_contig(uct_ib_mlx5_md_t *md,
-                                     uct_ib_mlx5_devx_mr_t *mr, off_t offset,
+                                     uct_ib_mlx5_devx_mem_t *memh,
+                                     uct_ib_mr_type_t mr_type,
```

Comment:
maybe pass mr_address as extra param (instead of memh+mr_type) since uct_ib_mlx5_devx_reg_ksm_data_contig is low-level function

Reply:
Done.